doc/manuals/installation_and_administration_manual/oauth2_hive_auth_provider.md

```diff
@@ -247,8 +250,28 @@ If the token exists but does not match the given user, then something like this
 2016-01-27 16:12:11,521 INFO  [pool-5-thread-32]: authprovider.HttpClientFactory (OAuth2AuthenticationProviderImpl.java:Authenticate(78)) - Response received: {"organizations": [], "displayName": "frb", "roles": [{"name": "provider", "id": "106"}], "app_id": "8556cc76154f41b3b43d7b31f0699982", "email": "frb@tid.es", "id": "frb"}
 2016-01-27 16:12:11,521 ERROR [pool-5-thread-32]: transport.TSaslTransport (TSaslTransport.java:open(296)) - SASL negotiation failure
 javax.security.sasl.SaslException: Error validating the login [Caused by javax.security.sasl.AuthenticationException: The given token does not match the given user]
+``` 
+
+[Top](#top)
+
+###<a name="section6.2"></a>Cache
+It is important to note this authentication provider caches OAuth2 tokens. Caching tokens is useful because it saves queries to the Identity Manager, and because it allows using expired tokens from clients not implementing the refreshing mechanism (e.g. Cygnus).
+
+This is done transparenty to the user, nevertheless from the administration point of view this is relevant regarding certain logs may appear, and the backup file for the cache.
+
+Logs are about cache usage, possibilities are:
+
+```
+17/03/02 09:14:20 INFO authprovider.OAuth2AuthenticationProviderImpl: User was not cached or token did not match, thus querying the IdM
+...
+17/03/02 09:14:20 INFO authprovider.OAuth2AuthenticationProviderImpl: User cached
+...
+17/03/02 09:14:29 INFO authprovider.OAuth2AuthenticationProviderImpl: User and token were cached, thus nothing to query to IdM
+...
 ```
 
+With regards to the backup file for the cache, this is a file saving the cache between runs of HiveServer2, loaded each time the server starts, and saved each time a modification is done in the cache. It is always saved as `/home/hive/oauth2.cache` file.
```

Comment:
The following sentence could be added to remark this is not changeable:

"This pathfaile is not configurable (it is hardwired in the code)"